tests/integration/telemetry/stackdriver/api/stackdriver_filter_test.go

```diff
@@ -95,7 +95,11 @@ func TestStackdriverMonitoring(t *testing.T) {
 }
 
 func TestMain(m *testing.M) {
+	// nolint: staticcheck
 	framework.NewSuite(m).
+		Skip("fake gcp metadata does not work anymore. see https://github.com/istio/istio/issues/35923#issuecomment-1302423155").
```

Comment:
Do we want to track this in a Jira? If yes, please link it here

Reply:
We used to, but let's not block this pr just for this.

Reply:
I'll add link to the commit in the issue so at least we can easily `git revert` it.